documentation/mqtt-overview.md

```diff
@@ -0,0 +1,83 @@
+
+The Broker responsible for receiving client-initiated connections and forwarding messages sent by the client to some other eligible clients.
+
+The Clients have the ability to subscribe to Topics in the Broker's directory to pull down any published updates (data or commands) from that point forward. They also have the ability to publish updates to any Topic the Broker gives them access to. 
+
+Clients are responsible for creating any instances of a connection to the Broker and thus the network is "edge-driven". Connections work over TCP and need only one outgoing port per connection instance for 2-way communications.
+
+The messages consists of only two core parts: the Topic and the **Payload**. While the topic is self-explanatory, the Payload isn't really formally defined by MQTT. It can be anything from a single raw bit to a JSON file.
+
+Here's an example of a simple MQTT connection that consists of two Clients (Client Ae and Client B), a Broker, and a single Topic of "network/facility/clientA/temp". In this example, this is a simple remote monitoring scenario for a temperature sensor where Client A acts as a publisher and Client be acts as a subscriber.
+
+```mermaid
+graph
+    A("[Broker] Topic: network/facility/clientA/temp") <-- "Publish: network/facility/clientA/temp" --> B(("Client A"))
+    A <-- "Subcribe: network/facility/clientA/temp" --> C(("Client B"))
+
+```
+
+In a more complex network, there is likely many Topics per Client and the Broker will be in charge of handling thousands of topics at once. Clients can also send commands to other clients on the network via Topics to trigger a real world action, such as a heater turning on or a status light changing color.
+
+## M5 Specifics
+
+### MQTT Deployment
+
+AnkerMake has deployed MQTT via [EMQX](https://www.emqx.com/en). Most likely it's [EMQX Cloud](https://www.emqx.com/en/cloud) since the servers are hosted by [AWS](https://aws.amazon.com/). We got that info via typing the following command into a terminal:
+
+```
+nslookup make-mqtt.ankermake.com
+```
+
+That returns something similar to:
+
+```
+Non-authoritative answer:
+Name:    ankermake-emqx-nlb-60dd062d3b52769a.elb.us-east-2.amazonaws.com
+Addresses:  3.22.247.85
+          3.134.119.140
+          18.189.81.10
+Aliases:  make-mqtt.ankermake.com
+```
+
+As you can see in the `Name:` field, it's an EMQX instance running on an AWS server.
+
+### Update Behavior
+
+As long as a M5 is powered on, it publishes updates to the broker servers under the `/phone/maker/YOUR_SERIAL_NUMBER_HERE/notice` topic as a client device. The update seems to be done by exception, meaning that it only updates when a value changes.
+
+The M5 also subscribes to  commands that may be sent to it from the MQTT servers for execution.
```

Comment:
I think we can explain this better.

The M5 also subscribes to other topics, where it will react to events, and treat them as commands to perform various actions. In this way, the M5 can be partially remote-controlled over MQTT.